spec/models/comment_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  before(:all) do
+    @user = User.create(name: 'test', photo: 'https://unsplash.com/photos/', bio: 'test')
+    @post = Post.create(author_id: @user.id, title: 'test', text: 'test', user_id: @user.id)
+  end
+
+  it 'should create a comment' do
+    comment = Comment.create(author_id: @user.id, post_id: @post.id, text: 'test', user_id: @user.id)
+    expect(comment).to be_valid
+  end
+
+  it 'should not create a comment without an author' do
+    comment = Comment.create(post_id: @post.id, text: 'test')
+    expect(comment).to_not be_valid
+  end
+end
```

Comment:
- [ ] These tests are failing. You need to remove `user_id: @user.id` on lines 6 and 10 as they are unnecessary. Neither Comment nor Post schemas have a `user_id` column.

  ![image](https://user-images.githubusercontent.com/54848363/213817180-a6136a8f-5352-4f27-8fab-e4cf326c718c.png)

- [ ] The first test will still fail, because of the `update_comments_counter` in the Comment model.
  
  ![image](https://user-images.githubusercontent.com/54848363/213817411-5543bd0b-cc3c-45f8-a263-fff1ced6dde8.png)

  You can fix this by using small letter `post`, instead of capital letter `Post`. `Post` refers to the table itself, but `post` refers to the actual post that the comment belongs to.

  ```
  def update_comments_counter
    post.increment!(:comments_counter)
  end
  ```